provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -148,22 +148,19 @@ def _add_pool(self, pool: PoolHandle, out_list: List[PoolDesc]) -> None:
         cvg_per_node = int(conf.get(
             f'server_node>{machine_id}>storage>cvg_count'))
 
-        cvg_per_storage_set = cvg_per_node * node_count
-
-        if layout.data + layout.parity + layout.spare == 0:
-            raise RuntimeError('All layout parameters are 0')
+        total_unit = layout.data + layout.parity + layout.spare
```

Comment:
let's move the code to calculate allowed_failures vector to a function. And add the algorithm as a comment as a function header.

Reply:
@mssawant Done